components/brave_ads/browser/prefs_util.cc

```diff
@@ -0,0 +1,33 @@
+/* Copyright (c) 2020 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#include "brave/components/brave_ads/browser/prefs_util.h"
+
+namespace brave_ads {
+namespace prefs {
+
+const base::Value* GetValue(
```

Comment:
not sure I understand the purpose of this method, but also it doesn't appear to be used anywhere?

Reply:
also seems a bit odd in brave_ads "public api" since there's nothing ad specific about it

Reply:
Default values for prefs are not persisted which caused an issue with ads during migration paths